src/components/JumpToPage.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {  
+  InputBase,  
+  MenuItem,  
+  Select,  
+  Toolbar,  
+  Typography
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { getPageValue } from '../utils.js';
+import classnames from 'classnames';
+
+const defaultStyles = theme => ({
+  root: {
+    color: theme.palette.text.primary,
+  },
+  caption: {
+    flexShrink: 0,
+  },
+  /*  Styles applied to the Select component root element */
+  selectRoot: {
+    marginRight: 32,
+    marginLeft: 8
+  },
+  select: {
+    paddingTop: 6,
+    paddingBottom: 7,
+    paddingLeft: 8,
+    paddingRight: 24,
+    textAlign: 'right',
+    textAlignLast: 'right',
+    fontSize: theme.typography.pxToRem(14),
+  },
+  /* Styles applied to Select component icon class */
+  selectIcon: {},
+  /* Styles applied to InputBase component */
+  input: {
+    color: 'inhert',
+    fontSize: 'inhert',
+    flexShrink: 0,
+  }
+});
+
+function JumpToPage(props) {
+
+  const handlePageChange = (event) => {
+    props.changePage( parseInt(event.target.value,10));
+  };
+
+  const {
+    count, 
+    classes,
+    textLabels,
+    rowsPerPage,
+    page,
+    changePage
+  } = props;
+
+  const textLabel = textLabels.pagination.jumpToPage;
+
+    let pages = [];
+  let lastPage = Math.min( 1000, getPageValue(count, rowsPerPage, 1000000));
```

Comment:
In the case of ServerSide, it seems that we need to find the value in another way.

For example, Add  `lastPage` props, options

Reply:
It'll still work. When using serverSide, you're supposed to have the total count of the table (but you only display the current page of entries). This feature is actually ported over from a fork I was using and I have a bunch of serverSide tables.